### ISSUE TITLE
DBUS_SESSION_BUS_ADDRESS

### ISSUE THREAD

**Issue by teohhanhui** [slp/krun#6]
Can we set that somehow? It seems to be used by Steam.

**Comment by SO5005** [comment]
dbus-launch sets it.

**Comment by teohhanhui** [comment]
Oh, I see... But apps inside the VM would not be able to talk to services on the host's session bus, right?

**Comment by SO5005** [comment]
dbus-launch isn't installed per default on fedora, dbus-x11 needs to be installed to get dbus-launch.

**Comment by SO5005** [comment]
If I run dbus-launch it sets DBUS_SESSION_BUS_ADDRESS to something different than it was before. (originally it was `unix:path=/run/user/1000/bus` dbus-launch sets it to something like `unix:path=/tmp/dbus-CB34ABmjaq,guid=2b0f4a14646900e5ac3fd40366470d34` but it does not export DBUS_SESSION_BUS_ADDRESS)

**Comment by slp** [comment]
The VM can't access the system's dbus. We could implement a unixsock proxy using vsock, but we'd need a strong justification for doing that. How's Steam using it? Is there a fallback behavior if it's not present?

**Comment by teohhanhui** [comment]
Not sure what Steam is using it for. There's a fallback but before that it's noisy in the log. Simply installing `dbus-x11` silences it.

**Comment by teohhanhui** [comment]
Yeah, will try to raise this with RPM Fusion. I think it should be considered a packaging bug.
